docs/installation/installing-with-npm.md

```diff
@@ -1,303 +1,196 @@
-
-```html
-    <script src="path-to-assets/govuk-frontend/govuk/all.js"></script>
-    <script>window.GOVUKFrontend.initAll()</script>
-  </body>
-</html>
+```javascript
+var path = require('path');
+app.use('/assets', express.static(path.join(__dirname, '/node_modules/govuk-frontend/govuk/assets')))
 ```
 
-#### Initialise GOV.UK Frontend in only certain sections of a page
-
-By default, the `initAll` function from GOV.UK Frontend initialises all components scoped to an entire page with the `document` object.
+#### Copy files
 
-You can change this by passing the `scope` parameter to the `initAll` function.
+Copy the following folders into your project folder:
 
-For example, if you have a modal dialog box that opens with new markup you could do the following:
-
-```js
-var $modal = document.querySelector('.modal')
-window.GOVUKFrontend.initAll({
-  scope: $modal
-})
-```
+- `/node_modules/govuk-frontend/govuk/assets/images`
+- `/node_modules/govuk-frontend/govuk/assets/fonts`
 
-#### Initialise individual included components
+You should use an automated task or your build pipeline to copy the files, so your project folder stays up to date when we update GOV.UK Frontend.
 
-GOV.UK Frontend components with JavaScript behaviour have the `data-module` attribute set in their markup.
+#### If you use different folders
 
-You can use this attribute to initialise the component manually. This may be useful if you are adding markup to a page after it has loaded.
+If you use different folders for images and the font, you can override GOV.UK Frontend's default paths with your paths.
 
-To initialise the first radio component on a page, use:
+If your `image` and `font` folders are together in the same folder, set the `$govuk-assets-path` variable to your folder. For example:
 
-```js
-var Radios = window.GOVUKFrontend.Radios
-var $radio = document.querySelector('[data-module="govuk-radios"]')
-if ($radio) {
-  new Radios($radio).init()
-}
+```SCSS
+$govuk-assets-path: '/{YOUR-FOLDER}';
+@import "govuk-frontend/govuk/all";
 ```
 
-### Option 2: Import JavaScript
-
-If you're using a bundler such as [Webpack](https://webpack.js.org/), use the `import` syntax to import all components. To initialise them, use the `initAll` function:
-
-```JS
-import { initAll } from 'govuk-frontend'
+If your `image` and `font` folders are in different folders, set the `$govuk-images-path` and `$govuk-fonts-path` variables to your folders. For example:
 
-initAll()
+```SCSS
+$govuk-images-path: "/{YOUR-IMAGES-FOLDER}/";
+$govuk-fonts-path: "/{YOUR-FONTS-FOLDER}/";
+@import "govuk-frontend/govuk/all";
 ```
 
-If you're using a bundler such as [Browserify](http://browserify.org/), you may need to use the CommonJS `require`:
+You can also use your own function to generate paths, for example if you're using `asset-pipeline` in [sass-rails](https://github.com/rails/sass-rails). Set the `$govuk-image-url-function` variable to the name of your function.
```

Comment:
I wonder if we should have an example of how to do this? Since we've now removed:
> See `src/govuk/settings/_assets.scss` for more information and examples.

Reply:
I wonder whether we need an example - the description seems fairly clear on its own, and I'm keen to avoid going too deep into advice about specific frameworks (at least until we've seen from research whether that's a major need). Could we leave as is and see if Rails users hit issues?

Reply:
Okay that sounds good 👍